### PR TITLE
Git practice

### PR DESCRIPTION
**Pull request by dasbeerboot**
This is description
I added 'hello world'

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -1,5 +1,6 @@
 body {background-color: #e6d4f7;
 }
+
```
Comment (dasbeerboot) [inline css/style.css:3]: Why did you change this line?

**Review by youngjoon-lee — CHANGES_REQUESTED** [review]
Fix it!

File: css/style.css (hunk 1 above)
Comment (dasbeerboot) [reply css/style.css:3]: No reason

**Review by youngjoon-lee — APPROVED** [review]
Good job!
